netci.groovy

```diff
@@ -184,4 +184,46 @@ Utilities.addGithubPRTrigger(prCCJob, 'Code Coverage Windows Debug', '@dotnet-bo
         Utilities.addStandardOptions(newJob)
         Utilities.addXUnitDotNETResults(newJob, 'bin/tests/**/testResults.xml')
     }
+
+// **************************
+// Outerloop and Innerloop against the latest dependencies on Windows. Rolling daily for debug and release
+// **************************
+
+['Debug', 'Release'].each { configuration ->
```

Comment:
Not Linux as well? Latest dependencies might work differently between Linux and Windows. 
Is having both Debug and Release valuable here? This is effectively just a sanity check. 

I'm thinking that having just a Linux Debug and Windows Debug is probably good enough?